templates/macros/assets.html

```diff
@@ -21,14 +8,14 @@
 </svg>
 {% endmacro license_logo_svg %}
 
-
 {% macro card(post) %}
 <div id="{{ post.title | slugify }}">
     <a class="asset-card" href="{{ post.extra.link }}">
```

Comment:
The [MDN docs for `a`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) state:

> Content within each `<a>` **should** indicate the link's destination.

Such that users with visual impairments/using screen-readers can tell where they are going.
I assume that this is not the case here, because we are wrapping a `div` element.

However, I could not find any guidelines to make this accessible. If anyone knows anything, please let me know!

Maybe it's fine though because of the card title that's deeper in the hierarchy?

Reply:
I think they mean that the content of the `<a>` should indicate where the link leads to: [Docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#strong_link_text)